impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -21,19 +22,22 @@ static TextureGLES::Type GetTextureTypeFromDescriptor(
   const auto usage = static_cast<TextureUsageMask>(desc.usage);
   const auto render_target =
       static_cast<TextureUsageMask>(TextureUsage::kRenderTarget);
-  if (usage == render_target) {
+  const auto is_msaa = desc.sample_count == SampleCount::kCount4;
+  if (usage == render_target && !is_msaa) {
+    // TODO(matanlurey): MSAA render buffers?
+    // See https://github.com/flutter/flutter/issues/137095.
     return TextureGLES::Type::kRenderBuffer;
```

Comment:
I thought we were just getting rid of `kRenderBuffer` altogether.

Reply:
We shouldn't try to mix refactors and feature work.